internal/commands/exec.go

```diff
@@ -22,8 +24,21 @@ type ExecOptions struct {
 	Command       []string
 	Task          string
 	ContainerName string
+	Explain       bool
 }
 
+var explainExecTmpl = `
+TASK_ID=$(aws ecs list-tasks --cluster {{.Env}}-{{.Namespace}} --service-name {{.Env}}-{{svc}} --desired-status "RUNNING" | jq -r '.taskArns[]' | cut -d'/' -f3 | head -n 1)
+
+aws ecs execute-command  \
+    --region {{.AwsRegion}} \
+    --cluster {{.Env}}-{{.Namespace}} \
+    --task $TASK_ID \
+    --container {{svc}} \
+    --command {{command}} \
+    --interactive
```

Comment:
Can we reorder maybe like this, so the command ends up in the end?
```suggestion
aws ecs execute-command  \
    --interactive \
    --region {{.AwsRegion}} \
    --cluster {{.Env}}-{{.Namespace}} \
    --task $TASK_ID \
    --container {{svc}} \
    --command {{command}}
```